### PR TITLE
json file configurated

### PR DESCRIPTION
**Pull request by guardar**

### CODE REVIEW

File: B827EBFFFEBFB1AF.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEBFB1AF",
+    "servers": [
+      {
+        "server_address": "eu.thethings.network",
```
Comment (gonzalocasas) [inline B827EBFFFEBFB1AF.json]: The server address is not valid. Please check [this list](https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway).

**Comment by guardar** [comment]
Server changed

**Comment by gonzalocasas** [comment]
But you closed it? You don't want to merge it?
Also, you selected the US router but according to your coords, you're in Spanish
Spain so you should use the European one.

(Also we can switch to Spanish if you prefer)

**Comment by guardar** [comment]
I'm sorry, I click it without thinking. Yes, I  want to merge.
Actually I'm in Spain, but I'm testing with equipment that uses the 915Mhz band because I'm thinking of installing it in the USA.
Thank you very much



________________________________
De: Gonzalo Casas <notifications@github.com>
Enviado: miércoles, 17 de enero de 2018 16:04
Para: ttn-zh/gateway-remote-config
Cc: guardar; State change
Asunto: Re: [ttn-zh/gateway-remote-config] json file configurated (#474)


But you closed it? You don't want to merge it?
Also, you selected the US router but according to your coords, you're in Spanish
Spain so you should use the European one.

(Also we can switch to Spanish if you prefer)

**Comment by gonzalocasas** [comment]
@guardar done!

**Pull request merged**